models/prices/prices_tokens.sql

```diff
@@ -170,7 +170,7 @@ VALUES
     ("tbc-terablock", "bnb", "TBC", "0x9798df2f5d213a872c787bd03b2b91f54d0d04a1", 18),
     ("sps-splintershards", "bnb", "SPS", "0x1633b7157e7638c4d6593436111bf125ee74703f", 18),
     ("dec-dark-energy-crystals", "bnb", "DEC", "0xe9d7023f2132d55cbd4ee1f78273cb7a3e74f10a", 3),
-
+    ("sfm-safemoon", "bnb", "SFM", "0x42981d0bfbaf196529376ee702f2a9eb9092fcb5", 9)
```

Comment:
@aalan3 is this missing a comma?

Reply:
Ah yeah I think so

Reply:
Should I do a new PR just adding the comma?

Reply:
yeah my bad!